content/en/logs/log_collection/java.md

```diff
@@ -319,54 +283,60 @@ It is possible to stream logs from your application to Datadog or to the Datadog
 There are two steps to configure the Java application to stream logs directly to Datadog:
 
 1. Add the Logback logging library to your code (or build a bridge from your current logger to it)
-2. Configure it to send logs to Datadog
+2. Configure Logback to send logs to Datadog
 
 ### Bridge from Java logging libraries to Logback
 
-* The recommended logging library to stream logs directly is Logback [logstash-logback-encoder][3].
-
-This logging library can be linked from the most common Java ones:
+Most common logging libraries can be bridged to Logback
```

Comment:
What does this mean, can you include more context? Also looks like this needs a period to be a complete statement.

Reply:
This is connected to the "There are two steps to configure the Java application to..." section right before.

How about:

```suggestion
If you are not using Logback as your logging library, the first step to stream logs directly to Datadog is to bridge your logging library to Logback. The exact steps depends on your logging library:
```